src/main/scala/viper/gobra/frontend/info/implementation/property/Addressability.scala

```diff
@@ -60,7 +61,7 @@ trait Addressability extends BaseProperty { this: TypeInfoImpl =>
           case t => Violation.violation(s"Expected slice, array, map, or sequence, but got $t")
         }
       case n: PDot => resolve(n) match {
-        case Some(s: ap.FieldSelection) => AddrMod.fieldLookup(addressability(s.base))
+        case Some(s: ap.FieldSelection) => AddrMod.fieldLookup(addressabilityMemberPath(addressability(s.base), s.path))
```

Comment:
why wasn't it necessary earlier to iterate over the path?

Reply:
That was a bug.